### PR TITLE
Added selected count on Mass Action 

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#7100

### CODE REVIEW

**Comment by Amitk-webkul** [comment]
@akathuria289 

If there is not any selected row so no need to show the selected title that is already shown. [Here](https://prnt.sc/mxa6ERGOcX7Y)

and No need to show selected tables where the table is not selectable for the data. [Here](https://prnt.sc/oWP_VMRWe4Xr)

**Comment by akathuria289** [comment]
@Amitk-webkul Done

**Comment by Amitk-webkul** [comment]
@akathuria289 
Now It is working fine.

**Comment by ashishkumar-webkul** [comment]
Checked and Found that this is working fine, I am attaching the video for the reference please check the same.
**Video**


https://user-images.githubusercontent.com/92721837/229710576-a4ed4ef6-258e-49fd-9fca-33a1d8a9dc67.mp4

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue [hunk 1]
```diff
@@ -107,7 +109,9 @@ export default {
         'massActionTargets',
         'dataId',
         'massActionsToggle',
-        'extraFilters'
+        'extraFilters',
+        "itemCount",
+        "total"
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue:114]: Single qoutes.

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue [hunk 2]
```diff
@@ -172,6 +175,7 @@ export default {
                 .then(function (response) {
                     if (response.status === 200) {
                         let results = response.data;
+                        console.log(results);
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue:178]: Remove this.

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue [hunk 3]
```diff
@@ -122,7 +124,8 @@ export default {
             massActionTargets: [],
             url: this.src,
             indexes: [],
-            massActionsToggle:false
+            massActionsToggle:false,
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue:127]:
```
massActionsToggle: false,
```

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue [hunk 4]
```diff
@@ -81,6 +81,8 @@
                 style="margin-left: 10px; white-space: nowrap;"
             ></button>
         </form>
+
+        <div class="item-count" v-if="itemCount"> {{ itemCount }} of {{ total }} selected</div>
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/partials/datagrid-filter-tags.vue:85]: Translation.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue (hunk 3 above)
Comment (akathuria289) [reply packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/datagrid-plus.vue]: added

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**